members/SARL35_283916.yaml

```diff
@@ -0,0 +1,54 @@
+bioguide: SARL35_283916
```

Comment:
Do you want to keep with the name comments at the top? I don't really care, just checking with you

Reply:
yeah, why not. Doesn't hurt